src/index.html

```diff
@@ -22,6 +22,333 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#moyoLogo"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo Company Logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <li>
```

Comment:
The `li` tag must be nested inside a parent `ul` or `ol` tag. Please, wrap your list items inside one of these tags.